test/hotspot/gtest/logging/test_asynclog.cpp

```diff
@@ -244,13 +245,34 @@ TEST_VM_F(AsyncLogTest, logBuffer) {
 }
 
 TEST_VM_F(AsyncLogTest, droppingMessage) {
-  if (AsyncLogWriter::instance() == nullptr) {
+  if (AsyncLogWriter::instance() == nullptr) return;
+  if (LogConfiguration::async_mode() != LogConfiguration::AsyncMode::Drop) {
+    EXPECT_TRUE(false) << "This test must be run in drop mode if async UL is activated";
```

Comment:
EXPECT_TRUE(false) looks a little weird but probably does what you want;  could you just use FAIL() here ?